resources/assets/js/components/ApplicationReview/ApplicationReview.tsx

```diff
@@ -348,19 +360,15 @@ class ApplicationReview extends React.Component<
               href={applicationUrl}
             >
               <i className="fas fa-file-alt" />
-              <FormattedMessage
-                id="apl.viewApplication"
-                defaultMessage="View Application"
-                description="Button text View Application"
-              />
+              {intl.formatMessage(messages.viewApplicationText)}
             </a>
             <a
               title={intl.formatMessage(messages.viewProfileTitle)}
               href={applicantUrl}
             >
               <i className="fas fa-user" />
               <FormattedMessage
-                id="apl.viewProfile"
+                id="application.review.viewProfile"
```

Comment:
Why did you change other instances of <FormattedMessage> to `intl.formatMessage()` but keep this one? Its fine, just inconsistent.

Reply:
Oops, missed this one.